tests/components/cert_expiry/test_config_flow.py

```diff
@@ -3,152 +3,235 @@
 import ssl
 from unittest.mock import patch
 
-import pytest
+from asynctest import patch as asyncpatch
```

Comment:
Please keep this named `patch`. We don't need `patch` from `unittest.mock` since `asynctest.patch` can handle both coroutines and normal functions.